ambari-infra-manager/pom.xml

```diff
@@ -400,7 +439,7 @@
     <dependency>
       <groupId>org.webjars</groupId>
       <artifactId>swagger-ui</artifactId>
-      <version>2.2.2</version>
+      <version>3.19.0</version>
```

Comment:
it worth a re-check how is that working with our swagger.html, as it should not have references to 2.2.2 js files anymore. maybe browser just cached those files etc.

Reply:
I added a commit which replace the swagger.html to the new version.